app/javascript/app/pages/ndc-sdg/ndc-sdg-styles.scss

```diff
@@ -1,13 +1,19 @@
 @import '~styles/layout.scss';
+@import '~styles/settings.scss';
 
-.grid {
-  @extend %grid;
+.bg {
+  background-color: $light-gray;
+}
 
-  @include msGridColumns(1fr, 2fr);
+.header {
+  padding-bottom: $gutter-padding;
+  overflow: auto;
 
-  grid-template-columns: 1fr 2fr;
-}
+  @media #{$mobile-only} {
+    height: 50vh;
+  }
 
-.bg {
-  background-color: $light-gray;
+  @media #{$tablet-portrait} {
+    height: 45vh;
```

Comment:
To match the map height size, a good idea could be to save it in a variable and share it across components, couldn't it?